src/tests/ftest/nvme/fault.yaml

```diff
@@ -59,10 +59,9 @@ ior:
   iorflags:
       flags: "-w -F -r -R -k -G 1"
   test_file: /testFile
-  repetitions: 1
   transfersize_blocksize:
     16M:
-      transfer_size: 16777216 #16M
+      transfer_size: 16M
```

Comment:
This whole section could be simplified
```
  transfer_size: 16M
  dfs_oclass: "RP_2G1"
  dfs_dir_oclass: "RP_2G1"
```
Also, there isn't a `block_size` specified, so IOR will fail

Reply:
No test is going to calculate the block size based on how much % of pool needs to be filled up. https://github.com/daos-stack/daos/blob/643ef3af2b8ebbd0368c77bc413c0ff8106e010c/src/tests/ftest/util/nvme_utils.py#L145

Reply:
Ah, thank you :)

Reply:
So just the simplification then, if you need to push again (optional)